### PR TITLE
Remove admin names from contact page since they change so frequently …

### PR DESCRIPTION
**Pull request by CJS7070**
…and are always outdated. And to remove my name since I'm still getting communications directly because of this page.

### CODE REVIEW

File: liquid/templates/contact.html [hunk 1]
```diff
@@ -27,14 +27,6 @@
     </p>
     
     <p><strong>System Administrators</strong><br>
```
Comment (ace-n) [inline liquid/templates/contact.html]: Remove the <br> at the end.

File: liquid/templates/contact.html [hunk 2]
```diff
@@ -27,14 +27,6 @@
...
-       Calvin Shirley,
-       Nathan Handler,
-       Adam Madsen,
-       Eric Sauer,
-       Milan Dasgupta,
-       Jake Bailey,
-       Sam Fu,
-       Luke Liu
        <br>
```
Comment (ace-n) [inline liquid/templates/contact.html]: Remove the <br> line

**Comment by ace-n** [comment]
LGTM, except for that first \<br\> on line 29.

@sskhandek This might be pedantic, but the other entries have names as well. Should we list a name or two here (for stylistic consistency && in case we need to contact a specific person on Admin), or are we OK with Admin not having a specific contact name unlike the others?

**Comment by CJS7070** [comment]
I removed the extra <br> but at least one of them needs to be there.

Admin is under a lot of fluctuation now and we will be discussing this in the coming semester. Right now, there are effectively no specific contact points. For the time being, it's best to leave it blank. The contact e-mail is what's important.

**Comment by ace-n** [comment]
Whoops, looks like you're right about the \<br\> thing - if @sskhandek (or any other member of Top4) signs off on this change, then I'll merge it.

tl;dr: Looks good, just need someone on Top4 to approve not having a name listed.

**Comment by sskhandek** [comment]
I approve. Thanks for running this by me.
